source/common/network/well_known_names.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
We are moving away from the well_known_names files. If possible can you remove this and just return the string directly? It probably is not needed anywhere other than where you return it.

Reply:
Oh, how about tests and where we want to know what is exactly configured?

Reply:
For tests just write out the string that you want. For knowing what is configured, in general we should be keying on capabilities, not names.

Reply:
Removed `well_known_names.h` file, and using string constant directly.